src/utils/blob-to-stream.js

```diff
@@ -0,0 +1,20 @@
+import {Readable} from 'stream';
+
+/* c8 ignore start */
+async function * read(blob) {
+	let position = 0;
+	while (position !== blob.size) {
+		const chunk = blob.slice(position, Math.min(blob.size, position + 524288));
```

Comment:
Can we add some explanation (comment or meaningful constant name) for this magic number? I understand it workaround for Node.js incomplete implementation, but we don't know how long this code will stay, so, let's make it a little bit more maintanable

Reply:
Don't actually remember where i got it from. it's exactly 512 KiB or a half MiB
My aim was to get something reasonable small that would fit nicely into any typed array so it would be the power of 2 and also dividable by 8.
and also make it fit nicely into one so called "pool size" of any optimal size allocation

Maybe it was too big?

looking at what slice size chrome uses it seems to be 64 KiB / slice
```js
(await new Blob(['s'.repeat(10000000)]).stream().getReader().read()).value.byteLength // 65536
// it is the same size as 
new BigUint64Array(8192).buffer.byteLength // 65536
// 8192 is the same size of nodes default pool size
// So it means 8192 * node's default buffer pool size... any correlation?
```
- FF slice size is only 4096 - many calls back and forth
- Nodes buffer default pool size is [8192](https://nodejs.org/api/buffer.html#buffer_class_property_buffer_poolsize)
- node streams default high watermark is set to 16 KiB.

don't know what it's but 65536 seems appealing to me. also one of those [tiles](https://www.csie.ntu.edu.tw/~b01902112/65536/) games

Reply:
64K sounds better for me, still with a comment that its Chome behavior

Reply:
i can replace it with 65536 as a constant with some comment above it